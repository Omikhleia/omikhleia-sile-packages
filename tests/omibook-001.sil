\begin[class=omibook]{document}
\nofolios
\noheaders
% Test case from the omibook customization.
% Q. Can I have indented paragraphs after section titles?

\section{Section One}

First pararaph should not be indented.

Next paragraphs are.

\style:redefine[name=sectioning:base, as=_secbase, inherit=true]{
\paragraph[indentafter=true]}

\section{Section Two}

First pararaph should now be indented.

Next paragraphs are too.

\end{document}
