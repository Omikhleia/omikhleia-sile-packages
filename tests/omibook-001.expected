Set paper size 	595.275597	841.8897729
Begin page
Mx 	59.5276
My 	105.6045
Set font 	Gentium Plus;12;800;;normal;;LTR
T	20 w=5.6309	(1)
Mx 	65.1584
T	17 w=2.7480	(.)
Mx 	70.5490
T	54 72 70 87 76 82 81 w=36.5391	(Section)
Mx 	110.3110
T	50 81 72 w=19.3652	(One)
Mx 	59.5276
My 	123.6045
Set font 	Gentium Plus;10;400;;normal;;LTR
T	41 76 85 86 87 w=18.7549	(First)
Mx 	80.9583
T	83 68 85 68 85 68 83 75 w=37.8027	(pararaph)
Mx 	121.4369
T	86 75 82 88 79 71 w=27.6172	(should)
Mx 	151.7299
T	81 82 87 w=13.9893	(not)
Mx 	168.3950
T	69 72 w=9.6973	(be)
Mx 	180.7681
T	76 81 71 72 81 87 72 71 w=36.8262	(indented)
Mx 	217.5943
T	17 w=2.2900	(.)
Mx 	72.0276
My 	135.6045
T	49 72 91 87 w=19.5801	(Next)
Mx 	94.2902
T	83 68 85 68 74 85 68 83 75 86 w=46.5430	(paragraphs)
Mx 	143.5157
T	68 85 72 w=13.1689	(are)
Mx 	156.6847
T	17 w=2.2900	(.)
Mx 	59.5276
My 	159.6045
Set font 	Gentium Plus;12;800;;normal;;LTR
T	21 w=5.6309	(2)
Mx 	65.1584
T	17 w=2.7480	(.)
Mx 	70.5490
T	54 72 70 87 76 82 81 w=36.5391	(Section)
Mx 	110.3108
T	55 90 82 w=21.1172	(Two)
Mx 	72.0276
My 	177.6045
Set font 	Gentium Plus;10;400;;normal;;LTR
T	41 76 85 86 87 w=18.7549	(First)
Mx 	93.4565
T	83 68 85 68 85 68 83 75 w=37.8027	(pararaph)
Mx 	133.9333
T	86 75 82 88 79 71 w=27.6172	(should)
Mx 	164.2246
T	81 82 90 w=17.6074	(now)
Mx 	184.5061
T	69 72 w=9.6973	(be)
Mx 	196.8774
T	76 81 71 72 81 87 72 71 w=36.8262	(indented)
Mx 	233.7036
T	17 w=2.2900	(.)
Mx 	72.0276
My 	189.6045
T	49 72 91 87 w=19.5801	(Next)
Mx 	94.2884
T	83 68 85 68 74 85 68 83 75 86 w=46.5430	(paragraphs)
Mx 	143.5122
T	68 85 72 w=13.1689	(are)
Mx 	159.3619
T	87 82 82 w=13.5010	(too)
Mx 	172.8629
T	17 w=2.2900	(.)
End page
Finish
